src/oscar/apps/partner/importers.py

```diff
@@ -155,9 +151,6 @@ def __init__(self, logger):
 
     @atomic_compat
     def handle(self, product_class_name, filepath):
-        product_class = ProductClass.objects.get(
-            name=product_class_name)
```

Comment:
In dieser Datei ist auch noch viel Produktklasse drin. Vielleicht kannst du noch vor den Produktklassen dieses Importer-Zeug komplett rausschmeißen. Das sollte ohne Probleme gehen.